c++/test/TestPredicatePushdown.cc

```diff
@@ -557,4 +557,103 @@ namespace orc {
       TestFirstStripeSelectedWithStripeStats(reader.get(), pos);
     }
   }
+
+  TEST(TestPredicatePushdown, testSeekOverEmptyPresentStream) {
```

Comment:
nit: the bug has nothing to do with predicate pushdown, we'd better move it out of here.

Reply:
I move the ut into the TestReader.cc file because the ParseError exception was not handled during the read operation.